windows-nsis/build-snapshot

```diff
@@ -12,6 +12,7 @@ get_full_path() {
 }
 
 GIT="${GIT:-git}"
+OPENVPN_VERSION="${OPENVPN_VERSION:-2.3_git}"; export OPENVPN_VERSION
```

Comment:
Having a default for OPENVPN_VERSION makes sense imho. Why do you export the variable, though?